blocks/edit/prose/index.js

```diff
@@ -80,8 +81,8 @@ export default function initProse({ editor, path }) {
 
   const ydoc = new Y.Doc();
 
-  const server = 'wss://collab.da.live';
-  const roomName = `https://admin.da.live${new URL(path).pathname}`;
+  const server = `${collabOrigin}`;
```

Comment:
nit:
```suggestion
  const server = collabOrigin;
```

Or just use collabOrigin for the roomName prefix.